packages/math-input/src/components/keypad/__tests__/keypad.test.tsx

```diff
@@ -155,6 +156,40 @@ describe("keypad", () => {
         expect(screen.getByTestId("TIMES")).toBeInTheDocument();
     });
 
+    it(`forces CDOT in locales that require it`, () => {
+        // Arrange
+        jest.spyOn(wbi18n, "getLocale").mockReturnValue("az");
+
+        // Act
+        render(
+            <Keypad
+                onClickKey={() => {}}
+                convertDotToTimes={true}
+                onAnalyticsEvent={async () => {}}
+            />,
+        );
+
+        // Assert
+        expect(screen.getByTestId("CDOT")).toBeInTheDocument();
```

Comment:
Tests updated 👍 In response to your other comment (which GH won't let me respond to directly):

> Nice. A question: is the numeric-input possibly affected (like does the validate on widget handle both symbols correctly)?

AFAIK only the Expression widget uses the "expression" keypad which is the only keypad to have the multiplication button. All other widgets use the "fraction" keypad I believe. This is an interesting point though: is `4 x 4` different than `4 * 4`? IIRC I believe content creators account for this, but I'm [confirming here](https://khanacademy.slack.com/archives/C01AZ9H8TTQ/p1702393715865769?thread_ts=1701964646.419179&cid=C01AZ9H8TTQ).

UPDATE: Charlie says this will not affect the correctness in Expression and I don't think NumericInput has multiplication.